plugins/org.eclipse.n4js/src/org/eclipse/n4js/scoping/members/FieldFactory.java

```diff
@@ -46,6 +46,7 @@ public boolean isValid() {
 	@Override
 	public TField create(String name) {
 		TField field = TypesFactory.eINSTANCE.createTField();
+		field.setComposed(true);
```

Comment:
I was thinking about that but wanted to set the flag asap + wanted to keep all initializations together in one place (per member type)